source/faq.rst

```diff
@@ -0,0 +1,109 @@
+.. _interfaces:
+
+FAQ
+===
+
+Is Credits a cryptocurrency?
+----------------------------
+No. Credits is not a cryptocurrency and has nothing to do with
+cryptocurrencies. Credits is a software framework for creation of
+applications based on Distributed Ledger Technology (DLT).
+
+Is there a public Credits token?
+--------------------------------
+No. Since Credits is not a cryptocurrency there is no underpinning token
+or anything of a kind.
+
+Is there a public Credits' blockchain?
+--------------------------------------
+No. Credits is not a public blockchain and is not supposed to be. Credits
+Core framework is designed to help developers of conventional end user
+applications do their work, i.e. build applications, without having to worry
+much about implications of DLT.
```

Comment:
Publicly, but not permissionlessly, I'll make that more clear here.
Credits can be public, but not meant to be permissionless, I'm explaining that later in this file.